crates/glaredb/src/pg_proxy.rs

```diff
@@ -85,7 +87,9 @@ impl PgProxy {
                         debug!("client connected (proxy)");
                         match handler.proxy_connection(inbound).await {
                             Ok(_) => debug!("client disconnected"),
-                            Err(e) => debug!(%e, "client disconnected with error."),
+                            Err(PgSrvError::Io(e)) if e.kind() == io::ErrorKind::UnexpectedEof =>
+                                trace!("client disconnected"),
```

Comment:
should the `Ok()` case also be a trace then? should we distinguish between the two?

Reply:
will add

Reply:
Downgraded debugs to trace, and used different messages in both cases (in case we need to distinguish).